pom.xml

```diff
@@ -526,7 +526,7 @@
     <netty.dev.tools.directory>${project.build.directory}/dev-tools</netty.dev.tools.directory>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <netty.build.version>30</netty.build.version>
+    <netty.build.version>29</netty.build.version>
```

Comment:
why does it go down ?

Reply:
Thanks, this was a leftover https://github.com/sirensolutions/netty/pull/29/commits/37d3d881ade426c8f7a5d9d7a50c4e47ab297ac3